webhook/webhook.go

```diff
@@ -148,13 +148,13 @@ func New(
 					logger.Errorw("failed to fetch secret", zap.Error(err))
 					return nil, nil
 				}
-
-				serverKey, ok := secret.Data[certresources.ServerKey]
+				sKey, sCert := certresources.GetSecretDataKeyNamesOrDefault()
```

Comment:
Are you creating custom webhooks binaries where you drop the certificate controller? 

If so it might be better to add these overrides to the `webhook.Options` and when they're empty default them to the keys in the `certificates/resources` package.

Reply:
Makes sense the env var thingy is a bit too intrusive.